common/src/test/java/org/conscrypt/java/security/WrappedECPrivateKey.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.conscrypt.java.security;
+
+import java.math.BigInteger;
+import java.security.interfaces.ECPrivateKey;
+import java.security.spec.ECParameterSpec;
+import java.util.Objects;
+
+class WrappedECPrivateKey implements ECPrivateKey {
```

Comment:
Key wrapping is a term used JCA for a different purpose, e.g. see WrappedKeyEntry, so maybe best avoided here.

As these are classes used for tests only and potentially we might add other customisations, maybe something like TestECPrivateKey would be ok.

Reply:
Good point, done.